records_mover/db/vertica/unloader.py

```diff
@@ -65,9 +68,6 @@ def unload(self,
             raise NotImplementedError('S3 currently required for Vertica bulk unload')
 
     def s3_available(self) -> bool:
-        if self.s3_temp_base_loc is None:
-            logger.info("Not attempting S3 export - SCRATCH_S3_URL not set")
-            return False
```

Comment:
This was sort of an awkward limitation - it resulted in us not being able to export to an S3 target when we didn't have a temporary bucket, even though we wouldn't need a temporary bucket!